Chinese/大话西游.md

```diff
@@ -88,6 +88,4 @@
 
 [little_prince](little_prince/this_is_a_story_about_a_boy.md)
 
-[陳奕迅](Eason_Chan/Eason_Chan.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.